src/qt/splashscreen.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include <qt/splashscreen.h>
 
+#include <qt/guiutil.h>
```

Comment:
The rationale for adding the shortcut here is that on every window with an enabled close button the shortcut should be applicable.

It could be worth it taking into consideration disabling the close button or removing the bar all together.